config/routes/claims.rb

```diff
@@ -14,7 +14,8 @@
   resources :schools, only: %i[index show] do
     scope module: :schools do
       resources :claims, except: %i[destroy] do
-        resources :mentors, only: %i[new create edit update], module: :claims
+        resource :mentors, only: %i[edit update], module: :claims
+        resources :mentors, only: %i[new create], module: :claims
```

Comment:
🤔 Do these still need to be here?

I think these create equal routes:

```ruby
resources :mentors, only: [:new, :create]
```

```
                         Prefix Verb  URI Pattern                                                     Controller#Action
    claims_school_claim_mentors POST  /schools/:school_id/claims/:claim_id/mentors(.:format)          claims/schools/claims/mentors#create
 new_claims_school_claim_mentor GET   /schools/:school_id/claims/:claim_id/mentors/new(.:format)      claims/schools/claims/mentors#new
```

```ruby
resource :mentors, only: [:new, :create]
```

```
                          Prefix Verb  URI Pattern                                                 Controller#Action
 new_claims_school_claim_mentors GET   /schools/:school_id/claims/:claim_id/mentors/new(.:format)  claims/schools/claims/mentors#new
     claims_school_claim_mentors POST  /schools/:school_id/claims/:claim_id/mentors(.:format)      claims/schools/claims/mentors#create
```

Reply:
Yeah, we don't need both, changed all to resource